pyincore/analyses/buildingeconloss/buildingeconloss.py

```diff
@@ -30,36 +30,49 @@ def __init__(self, incore_client):
 
     def run(self):
         """Executes building economic damage analysis."""
+
         # Get inflation input in %
         self.infl_factor = self.get_parameter("inflation_factor")
```

Comment:
If you don't set an inflation value, I get the following error:

File "/home/cnavarro/git/pyincore/venv/lib/python3.8/site-packages/pyincore-1.2.0-py3.8.egg/pyincore/analyses/buildingeconloss/buildingeconloss.py", line 90, in get_inflation_mult
    return (self.infl_factor / 100.0) + 1.0
TypeError: unsupported operand type(s) for /: 'NoneType' and 'float'

Reply:
Done